demo-app/src/main/java/com/seatgeek/placesautocompletedemo/PlacesAutocompleteActivity.java

```diff
@@ -89,4 +91,20 @@ public void onFailure(final Throwable failure) {
             }
         });
     }
+
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+        getMenuInflater().inflate(R.menu.menu_main, menu);
+        return true;
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        int id = item.getItemId();
+        if (id == R.id.action_hide_x)
```

Comment:
Nit: prefer to use `{` here instead of shorthand. we try to follow google's java style guide

Reply:
Fixed